src/modules/project/pages1/projectView/views/rewards/hooks/useProjectRewardForm.ts

```diff
@@ -0,0 +1,313 @@
+import { gql, useQuery } from '@apollo/client'
+import { yupResolver } from '@hookform/resolvers/yup'
+import { format } from 'date-fns'
+import { ChangeEvent, useEffect, useState } from 'react'
+import { useForm } from 'react-hook-form'
+import { useNavigate } from 'react-router'
+import * as yup from 'yup'
+
+import { useBTCConverter } from '@/helpers'
+import { useProjectRewardsAPI } from '@/modules/project/API/useProjectRewardsAPI'
+import { useProjectAtom } from '@/modules/project/hooks/useProjectAtom'
+import { PathName } from '@/shared/constants'
+import { useModal } from '@/shared/hooks'
+import {
+  CreateProjectRewardInput,
+  ProjectRewardFragment,
+  RewardCurrency,
+  Satoshis,
+  UpdateProjectRewardInput,
+  USDCents,
+  useProjectRewardCurrencyUpdateMutation,
+  useProjectRewardQuery,
+} from '@/types'
+import { useNotification } from '@/utils'
+
+type FormValues = Omit<ProjectRewardFragment, 'id' | 'sold'>
+
+const MAX_REWARD_IMAGES = 5
+
+const REWARD_CATEGORIES_QUERY = gql`
```

Comment:
let's define queries in the graphql folder, it generates types and stays consistent with the rest.